hw2/05-form.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 // Add your code here
+const form = document.getElementById('regForm');
+
+const handleSubmit = function handleFormSubmit(event) {
+  const name = form.elements.name.value ? form.elements.name.value : 'no submission';
+  const email = form.elements.email.value ? form.elements.email.value : 'no submission';
+  const regStatus = form.elements.regStatus.value ? form.elements.regStatus.value : 'no submission';
+  const checkedCourses = form.querySelectorAll('input[name="course"]:checked');
+  const anythingElseText = form.elements.textField.value ? form.elements.textField.value : 'no submission';
+  let coursesStr = '';
+  if (checkedCourses.length === 0) {
+    coursesStr = 'no submission';
+  } else {
+    for (let i = 0, n = checkedCourses.length; i < n; i += 1) {
+      coursesStr += `, ${checkedCourses[i].value}`;
+    }
```

Comment:
Refactor this to use JavaScript's higher-order functions instead of a traditional for loop. This is the preference of the AirBnB style guide 11.1 [read here](https://github.com/airbnb/javascript#iterators--nope)